ddtrace/llmobs/_integrations/crewai.py

```diff
@@ -198,6 +197,57 @@ def _llmobs_set_tags_tool(self, span, args, kwargs, response):
             return
         span._set_ctx_item(OUTPUT_VALUE, response)
 
+    def _tag_agent_manifest(self, span, agent):
+        if not agent:
+            return
+
+        manifest = {}
+        manifest["framework"] = "CrewAI"
+        if hasattr(agent, "role"):
+            manifest["name"] = agent.role or "CrewAI Agent"
```

Comment:
```suggestion
        manifest["name"] = getattr(agent, "role", "CrewAI Agent")
```
i think as the logic is currently if there is no role then we don't set the name field of the manifest at all

Reply:
ah good suggestion!